README.md

```diff
@@ -1,10 +1,39 @@
 [![BCH compliance](https://bettercodehub.com/edge/badge/legalthings/lto-network-frontend?branch=master)](https://bettercodehub.com/)
 
-# LtoNetworkFrontend
+# LTO Network Frontend
+<a href="https://github.com/legalthings/lto-network-frontend/"><img src="https://lto.network/img/meta.png"></a>
+<!-- This project was generated with [Angular CLI](https://github.com/angular/angular-cli) using [Nrwl Nx](https://nrwl.io/nx).  -->
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) using [Nrwl Nx](https://nrwl.io/nx).
+## The Goal of Angular CLI
```

Comment:
This is not a goal of angular CLI. I think it would be better to name it as "Repository content".